itests/qtest/pom.xml

```diff
@@ -477,6 +477,23 @@
   </profiles>
   <build>
     <plugins>
+      <plugin>
+        <groupId>com.googlecode.maven-download-plugin</groupId>
+        <artifactId>download-maven-plugin</artifactId>
+        <version>1.3.0</version>
+        <executions>
+          <execution>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>wget</goal>
+            </goals>
+            <configuration>
+              <url>http://downloads.mariadb.com/Connectors/java/connector-java-2.2.0/mariadb-java-client-2.2.0.jar</url>
+              <outputFileName>mariadb-java-client-2.2.0.jar</outputFileName>
```

Comment:
I think it would be nice to not hard-code the versions of connectors. Can we use place holders instead? check above on how we manage versions for dependencies.

Reply:
My main concern for using a place holder instead of hard-code is we have the wget url, I am afraid that different version may have complication around the wget url pattern? right now hive metastore init test is also using the hard-coded version for that reason. So I think it may be better to leave it as it is for now?